app/assets/stylesheets/modules/org.css

```
@@ -265,3 +265,20 @@
     float: right; }
   .about__assets__download:focus:before, .about__assets__download:hover:before {
     animation: arrow .75s infinite; }
+
+/* Banner */
+
+#banner {
+  background-color: #141c22;
+  text-align: center;
+  padding: 12px 0;
```

Per [Jenny's note](https://github.com/rubygems/rubygems.org/pull/3945#issuecomment-1641053570) re: lack of padding, let's add horizontal padding here.

Recommending `15px` all around. Skimming through the spacing-related rules in `home.css`, it looks like they use multiples of 5s (look at the margin and padding rules).

```suggestion
  padding: 15px;
```